### PR TITLE
Create Milestone

### PR DESCRIPTION
**Pull request by benibenj**
When clicking on add milestone, a dropdown appears with all milestones. At the top right there is a plus which will create an inputbox where you can give the name of the new milestone. If you have already entered it and click on the plus (add), the inputbox appears with the text prefilled.

### CODE REVIEW

**Review by alexr00 — CHANGES_REQUESTED** [review]
Looks good, just a few questions.

File: src/github/pullRequestOverview.ts [hunk 1]
```diff
@@ -500,19 +500,47 @@ export class PullRequestOverviewPanel extends IssueOverviewPanel<PullRequestMode
...
+				const inputBox = vscode.window.createInputBox();
+				inputBox.title = 'Create new milestone';
+				inputBox.placeholder = 'New milestone name';
+				if (quickPick.value !== ''){
+					inputBox.value = quickPick.value;
+				}
+				inputBox.show();
+				inputBox.onDidAccept(() => {
+					if (inputBox.value !== ''){
+						this._folderRepositoryManager.createMilestone(this._item.githubRepository, inputBox.value);
```
[the remark this answers is not in the document]
Comment (benibenj) [reply src/github/pullRequestOverview.ts:522]: I was thinking about this. On the one hand you select to create a milestone so it might only do that as it's currently the only way to create a milestone, on the other hand, the user clicked on add milestone so it might as well just add it. I'll change it so it adds the milestone after creating it if you agree.

[the remark this answers is not in the document]
Comment (alexr00) [reply src/github/pullRequestOverview.ts:522]: Given that the user created the milestone while trying to assign a milestone to a PR, I agree that it makes sense to add the milestone after it's been created.

File: src/github/pullRequestOverview.ts [hunk 2]
```diff
@@ -500,24 +500,57 @@ export class PullRequestOverviewPanel extends IssueOverviewPanel<PullRequestMode
...
+						if (milestone !== undefined) {
+							await this.updateMilestone(milestone, message);
+						}
+					}
 				});
-			}
+			});
+
+			quickPick.show();
+			quickPick.items = await getMilestoneOptions(this._folderRepositoryManager);
```
Comment (alexr00) [inline src/github/pullRequestOverview.ts:532]: Just curious: why did you remove the `quickPick.busy` setting?

**Review by alexr00 — APPROVED** [review]
Looks good!
